### DISCUSSION TITLE
How to create WhatsApp channel on Baileys?

### DISCUSSION THREAD

**Discussion by JunaGanz** [WhiskeySockets/Baileys#432] [General]
can Baileys make A WhatsApp channel?

**Comment by multcriacoes** [comment]
and how can we send message to a channel?

**Comment by shizothetechie** [comment]
> can Baileys make A WhatsApp channel?
> 

Till now (17 December 2023)
Creating WhatsApp Channel using Baileys is not possible as Channel features are not added in baileys as this is new features but surely This will available in future :)
